docs/src/main/sphinx/release/release-417.md

```diff
@@ -0,0 +1,40 @@
+# Release 417 (10 May 2023)
+
+## General
+
+* Improve performance of `UNION ALL` queries. ({issue}`17265`)
+* Fix over-reading data in the `TEXTFILE` decoder for structs that contain more
+  columns in the file than are mapped in the schema. ({issue}`17364`)
+
+## Delta Lake connector
+
+* Add support for [`COMMENT ON VIEW`](/sql/comment). ({issue}`17089`)
+* Improve efficiency of packing data in files written by the Parquet writer. ({issue}`17373`)
+* Improve the layout of data in Parquet files, leading to faster reads. ({issue}`17404`)
+* Improve read performance for tables with `ROW` or `STRUCT` columns when only a
+  subset of fields is needed for a query. ({issue}`17085`)
+
+## Hive connector
+
+* Add the `extra_properties` table property to support adding arbitrary
```

Comment:
Add support for specifying arbitrary table properties via the `extra_properties` table property.